client/src/entry/analysis/menu.js

```diff
@@ -105,6 +105,7 @@ export function fetchMenu(options = {}) {
                 {
                     title: _l("Datasets"),
                     url: "/datasets/list_published",
+                    hidden: !Galaxy.user.id,
                 },
```

Comment:
```suggestion
```

Reply:
there is no grid for public datasets, this url was added erroneously during the transition under "Data" in masthead

Reply:
I think you're right, I'll merge and open a followup